base.d.ts

```diff
@@ -169,6 +171,33 @@ export type ParseOptions = {
 	```
 	*/
 	readonly parseFragmentIdentifier?: boolean;
+
+	/**
+	Specify a pre-defined schema to be used when parsing values.
+	Use this feature to override configuration options: parseNumber, parseBooleans, and arrayFormat.
+	pre-defined types specified here will be used even if parsing options such parseNumber as are not enabled.
```

Comment:
Don't hard-wrap.

It's also not clear to users how it affects `arrayFormat`.

Reply:
<img width="788" alt="Screenshot 2024-06-06 at 00 35 35" src="https://github.com/sindresorhus/query-string/assets/27223098/cb3d202d-3f3f-418e-98ed-5d900ea26359">

Good spot. I've left some line breaks for readability, but let me know your thoughts on this second pass!